src/web-stack/apps/api/src/app/users/entities/reaffirmation.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { AffirmationEntity } from './affirmation.entity';
+import { UserEntity } from './user.entity';
+
+/**
+ * TODO
+ * - Notes, fonts, stamps
+ * - Constructor
+ * - Time created 
+ * - Move interfaces to it's own directory
+ */
+
+// PLACEHOLDERS
+interface INote {
```

Comment:
Let's move INote, IFont, and IStamp to entities. This will allow us to add/remove fonts, notes, and stamps at will, especially later.